src/components/PolarisTestProvider/PolarisTestProvider.tsx

```diff
@@ -77,14 +77,12 @@ export function PolarisTestProvider({
 
   const uniqueIdFactory = new UniqueIdFactory(globalIdGeneratorFactory);
 
-  const features = {newDesignLanguage: false, ...featuresProp};
+  const features = {...featuresProp};
```

Comment:
No need to spread anymore and create a new object anymore. We can remove this declaration and avoid renaming the features prop above

```diff
- features: featuresProp = {},
+ features = {}
```